pkg/scheduler/framework/runtime/framework.go

```diff
@@ -315,21 +320,21 @@ func NewFramework(ctx context.Context, r Registry, profile *config.KubeScheduler
 		if err != nil {
 			return nil, fmt.Errorf("initializing plugin %q: %w", name, err)
 		}
-		pluginsMap[name] = p
+		f.pluginsMap[name] = p
 
 		f.fillEnqueueExtensions(p)
 	}
 
 	// initialize plugins per individual extension points
 	for _, e := range f.getExtensionPoints(profile.Plugins) {
-		if err := updatePluginList(e.slicePtr, *e.plugins, pluginsMap); err != nil {
+		if err := updatePluginList(e.slicePtr, *e.plugins, f.pluginsMap); err != nil {
```

Comment:
given `pluginsMap` is a private variable of `f`, maybe you can eliminate pluginsMap from the function signature of `updatePluginList()`? (same for `expandMultiPointPlugins()`)

Reply:
Thanks, fixed it.
BTW `updatePluginList` is not a method of `frameworkImpl` and doesn't include frameworkImpl as an argument, therefore I don't change the signature of `updatePluginList`.

Reply:
yup, a typo. I think I meant `getScoreWeights`